Source/aweXpect.Reflection/ThatMethods.Return.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using aweXpect.Core;
+using aweXpect.Core.Constraints;
+using aweXpect.Reflection.Helpers;
+using aweXpect.Results;
+
+// ReSharper disable PossibleMultipleEnumeration
+
+namespace aweXpect.Reflection;
+
+public static partial class ThatMethods
```

Comment:
Please add the same also for a single method in `ThatMethod` (name it there `Returns` instead of `Return` to be grammatically correct)